app/routes/schemas/enrich-request.js

```diff
@@ -8,7 +8,7 @@ const {
 module.exports = Joi.object({
   day: Joi.number().integer().min(1).max(31).required(),
   month: Joi.number().integer().min(1).max(12).required(),
-  year: Joi.number().integer().min(2020).max(9999).required(),
+  year: Joi.number().integer().min(2021).max(9999).required(),
   'debt-type': Joi.string().valid(ADMINISTRATIVE, IRREGULAR).required(),
   'invoice-number': Joi.string().required()
 })
```

Comment:
This should follow the same pattern as implemented throughout this repo, i.e. define individual schemas for each component (day, month, year, debt-type and invoice-number) with customer defined error codes and import them into page routes as individual use-cases, i.e. `enrich-request`, `capture-debt`, etc (add a tech debt ticket if 1 already doesn't exist)

Also, we should standardise which Joi functions we use, i.e. here we're using `min`, `max` but else where for ints we use `greater` and `less` -- which is more intuitive? Again another tech debt ticket

Reply:
added tickets to backlog